pkgs/applications/misc/dump1090/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, gnumake, pkgconfig, libusb, rtl-sdr, lib }:
+
+stdenv.mkDerivation rec {
+  name = "dump1090-${version}";
+  version = "2014-10-31";
+
+  src = fetchFromGitHub {
+    owner = "MalcolmRobb";
+    repo = "dump1090";
+    rev = "bff92c4ad772a0a8d433f788d39dae97e00e4dbe";
+    sha256 = "06aaj9gpz5v4qzvnp8xf18wdfclp0jvn3hflls79ly46gz2dh9hy";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ gnumake libusb rtl-sdr ];
```

Comment:
I take it that the output captures a reference to make that it wants to execute at runtime? Otherwise it's better to rely on gnumake being part of the standard build env.

Reply:
Ah, good point.  I have dropped `gnumake` and updated the PR.